### PR TITLE
Forecast request

### PR DESCRIPTION
**Pull request by vladyslav-babych**
- Added forecast request
- Refactored all strings to be used from resources

### CODE REVIEW

File: app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt [hunk 1]
```diff
@@ -11,4 +25,30 @@ fun CurrentWeatherResponse.toCurrentWeather(): CurrentWeather {
...
+            }
+        )
+    }
+}
+
+fun Hour.toHourly(): HourlyWeather {
+    return HourlyWeather(
+        hourlyTemp = temp,
+        hourlyConditionIcon = "https:${condition.conditionIcon}",
+        time = time.drop(11)
```
Comment (vladyslav-babych) [inline app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt]: Is this right way to format time like this?
Time pattern which we receive from API is `"yyyy-MM-dd HH:mm"`, I tried to format it to `"HH:mm"`, but unsuccessfully. So I decided to just drop first 11 symbols from received string (e.g. `"2022-09-13 17:00"`) and as a result we have a string `"17:00"`

Comment (RuslanKharkevych) [reply app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt]: This is definitely not a right way. You must use date pattern

File: app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt [hunk 2]
```diff
@@ -11,4 +25,30 @@ fun CurrentWeatherResponse.toCurrentWeather(): CurrentWeather {
...
+@RequiresApi(Build.VERSION_CODES.O)
+fun ForecastResponse.toForecast(): List<ForecastWeather> {
+    return forecast.forecastDay.map { item ->
+        val localDate = LocalDate.parse(item.date)
+        val month = localDate.month.toString()
+        val day = localDate.dayOfMonth.toString()
+
+        ForecastWeather(
+            dayTemp = item.day.dayTemp,
+            date = "$day $month",
```
Comment (RuslanKharkevych) [inline app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt]: Here you also should use pattern to format date

File: app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt [hunk 3]
```diff
@@ -11,4 +25,30 @@ fun CurrentWeatherResponse.toCurrentWeather(): CurrentWeather {
         conditionIcon = "https:${current.condition.iconCondition}",
         conditionText = current.condition.textCondition
     )
+}
+
+@RequiresApi(Build.VERSION_CODES.O)
```
Comment (RuslanKharkevych) [inline app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt]: This is not a good idea. This means that this function won't work on versions prior to Android 8

**Review by RuslanKharkevych — CHANGES_REQUESTED** [review]

**Review by RuslanKharkevych — APPROVED** [review]

**Pull request merged**